dockerscripts/download-static-curl.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+function download_arch_specific_executable {
+    curl -f -L -s -q \
+        https://github.com/moparisthebest/static-curl/releases/latest/download/curl-$1 \
+        -o /go/bin/curl \
+        || exit 1
+    chmod +x /go/bin/curl
+}
+
+case $TARGETARCH in
+    "arm64")
```

Comment:
Where is amd64?

Reply:
`amd64` and `ppc64le` are handled by the default case `*)`.

- `arm64` is special-cased because the `static-curl` project uses the `aarch64` terminology.
- `s390x` is special-cased because it is not supported by `static-curl`.